engines/scumm/he/moonbase/ai_main.cpp

```diff
@@ -20,14 +20,16 @@
  *
  */
 
-#include "scumm/he/intern_he.h"
-
-#include "scumm/he/moonbase/moonbase.h"
 #include "scumm/he/moonbase/ai_main.h"
-#include "scumm/he/moonbase/ai_traveller.h"
+
+#include "scumm/he/intern_he.h"
+#include "scumm/he/moonbase/ai_pattern.h"
 #include "scumm/he/moonbase/ai_targetacquisition.h"
+#include "scumm/he/moonbase/ai_traveller.h"
 #include "scumm/he/moonbase/ai_types.h"
-#include "scumm/he/moonbase/ai_pattern.h"
+#include "scumm/he/moonbase/moonbase.h"
+
+#include "common/math.h"
```

Comment:
Same comment as for fullpipe: I don't really see a reason to change the order of the includes.